examples/guestbook/README.md

```diff
@@ -110,14 +121,18 @@ redis-master        name=redis-master   name=redis-master
 ```
 
 
-This will cause all pods to see the redis master apparently running on <ip>:6379.
+This will cause all pods to see the redis master apparently running on <ip>:6379.  The traffic flow from slaves to masters can be described in three steps, like so.
+
+- a *redis slave* will connect to "port" on the *redis master service*
+- traffic will be forwarded from the service "port" to the  *containerPort* on the bound pod (i.e. the redis-master pod that is running).
+- finally, the *redis-master* pod will receive traffic on *hostPort* which it exposed, and send that to its *containerPort*.
```

Comment:
Can you remove the third step ? *redis-master* receives traffic on the *containerPort* as you've noted in the second step.